README.md

```diff
@@ -14,50 +14,49 @@ _Create a site or blog from your GitHub repositories with GitHub Pages._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
```

Comment:
<<< Author notes:  Course Start >>> This course really has helped me understand GitHub better.